tests/run_lattice_test.cpp

```diff
@@ -17,13 +17,10 @@
 
 #include <iostream>
 
-#include "test_AtomicMaxLattice.h"
-#include "test_BoolLattice.h"
-#include "test_MapLattice.h"
-#include "test_MaxLattice.h"
-#include "test_MinLattice.h"
-#include "test_SetLattice.h"
-#include "test_TombstoneLattice.h"
+#include "test_bool_lattice.hpp"
```

Comment:
We probably want to have the test directory structure match the source structure, so the lattice tests should be in `include/lattices`.